crates/bevy_camera/src/camera.rs

```diff
@@ -699,17 +699,19 @@ impl Camera {
     /// # Panics
     ///
     /// Will panic if the `camera_transform` contains `NAN` and the `glam_assert` feature is enabled.
-    pub fn world_to_ndc(
+    pub fn world_to_ndc<V: Into<Vec3A> + From<Vec3A>>(
         &self,
         camera_transform: &GlobalTransform,
-        world_position: Vec3,
-    ) -> Option<Vec3> {
+        world_point: V,
+    ) -> Option<V> {
         // Build a transformation matrix to convert from world space to NDC using camera data
-        let clip_from_world: Mat4 =
-            self.computed.clip_from_view * camera_transform.to_matrix().inverse();
-        let ndc_space_coords: Vec3 = clip_from_world.project_point3(world_position);
+        let view_point = camera_transform
+            .affine()
+            .inverse()
```

Comment:
Doesn't really need to happen, but it might be nice to have a separate named variable for what is `affine().inverse()`? That would be world_to_view I guess?